core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/io/caches/TextAnnotationMapDBHandler.java

```diff
@@ -30,17 +33,28 @@
  */
 public class TextAnnotationMapDBHandler implements TextAnnotationCache {
     private DB db;
+    private Logger logger = LoggerFactory.getLogger(TextAnnotationMapDBHandler.class);
 
     public TextAnnotationMapDBHandler(String dbFile) {
-        db = DBMaker.fileDB(dbFile).closeOnJvmShutdown().make();
+        try {
+            db = DBMaker.fileDB(dbFile).closeOnJvmShutdown().make();
+        }
+        catch (DBException e) {
+            logger.warn("mapdb couldn't instantiate db using file '{}': check error and either remove lock, " +
+                    "repair file, or delete file.", dbFile);
+            e.printStackTrace();
+            System.err.println("mapdb couldn't instantiate db using file '" + dbFile +
```

Comment:
dropped logger.warn() instead; I think of this as cause to exit at present (hence stderr) but it's possible a client could instead delete the offending directory and restart (in which case they can log).  If the mapdb cache is large I assume they may want to try other solutions first.